routes/web.php

```diff
@@ -13,8 +16,23 @@
 |
 */
 
+/* Routes for authentication and email verification */
 Auth::routes();
 
+Route::get('/email/verify', function () {
+    return view('auth.verify');
+})->middleware('auth')->name('verification.notice');
+
+Route::get('/email/verify/{id}/{hash}', function (EmailVerificationRequest $request) {
+    $request->fulfill();
+    return redirect('/home');
+})->middleware(['auth', 'signed'])->name('verification.verify');
+
+Route::post('/email/verification-notification', function (Request $request) {
+    $request->user()->sendEmailVerificationNotification();
+    return back()->with('message', 'Verification link sent!');
+})->middleware(['auth', 'throttle:6,1'])->name('verification.resend');
```

Comment:
@RheaMars gibt es einen Grund, dass du die Routes manuell deklariert hast? 

Ich würde Zeile 22 bis 34 entfernen, sowie die Zeilen drei und vier. 

```php
Auth::routes(['verify' => true]);
```

Reply:
Ich hatte tatsächlich keine Ahnung, dass das auch so einfach geht... :-D Kein Wort davon in der Doku: https://laravel.com/docs/8.x/verification

Reply:
Geh mal auf die Version 7.x
Da steht es noch drin. Frag mich nicht, warum die des herausgenommen haben :P